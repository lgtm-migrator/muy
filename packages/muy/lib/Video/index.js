import React from "react"
import Box from "@material-ui/core/Box"
import clsx from "clsx"

const Video = ({ autoPlay, classes, className, loop, muted, ...props }) => (
  <Box
    autoPlay={autoPlay}
    className={clsx(classes, className)}
    component={"video"}
    loop={loop}
    muted={muted}
    {...props}
  />
)

export default Video
