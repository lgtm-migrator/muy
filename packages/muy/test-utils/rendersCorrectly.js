import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
