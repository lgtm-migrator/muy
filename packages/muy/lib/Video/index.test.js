import React from "react"
import rendersCorrectly from "../../test-utils/rendersCorrectly"
import Video from "./index"

describe("<Video />", () => {
  it("should render correctly", () => {
    rendersCorrectly(<Video />)
  })
})
