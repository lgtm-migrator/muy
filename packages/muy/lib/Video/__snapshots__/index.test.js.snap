// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Video /> should render correctly 1`] = `
<video
  className="MuiBox-root MuiBox-root-2"
/>
`;
